java/core/pom.xml

```diff
@@ -134,7 +134,12 @@
         <dependency>
             <groupId>com.github.bbottema</groupId>
             <artifactId>emailaddress-rfc2822</artifactId>
-            <version>1.1.3</version>
+            <version>2.3.1</version>
+        </dependency>
+        <dependency> <!-- for email validation to work in lib above; used in RESTApiSanitizerImpl, PseudonymizerImpl as of June 2023 -->
+            <groupId>com.sun.mail</groupId>
+            <artifactId>jakarta.mail</artifactId> <!-- CDDL + GPLv2 with classpath exception license, so ideally would remove -->
```

Comment:
I think it should be fine, right? the classpath exception allows distribution as long as source code is available